ee/audit/audit_ee.go

```diff
@@ -130,19 +130,12 @@ func InitAuditor(conf *x.LoggerConf, gId, nId uint64) error {
 // trackIfEEValid tracks enterprise license of the cluster.
 // Right now alpha doesn't know about the enterprise/licence.
 // That's why we needed to track if the current node is part of enterprise edition cluster
-func trackIfEEValid(conf *x.LoggerConf, eeEnabledFunc func() bool) {
+func trackIfEEValid(conf *x.LoggerConf) {
 	defer auditor.closer.Done()
 	var err error
 	for {
 		select {
 		case <-auditor.tick.C:
-			if !eeEnabledFunc() && atomic.CompareAndSwapUint32(&auditEnabled, 1, 0) {
```

Comment:
we might still need the if condition here

Reply:
we can't change the audit log enable disable while alpha is running